ttrss/.env-dist

```diff
@@ -28,3 +28,10 @@ TTRSS_OAUTH2=
 # by entering which authorization group can log into your app. You create
 # groups of email addresses in the `traefik` folder by running `make groups`.
 TTRSS_OAUTH2_AUTHORIZED_GROUP=
+
+# Mutual TLS (mTLS):
+# Set true or false. If true, all clients must present a certificate signed by Step-CA:
+TTRSS_MTLS_AUTH=false
+# Enter a comma separated list of client domains allowed to connect via mTLS.
+# Wildcards are allowed and encouraged on a per-app basis:
+TTRSS_MTLS_AUTHORIZED_CERTS=*.clients.whoami.example.com
```

Comment:
change whoami to ttrss

Reply:
Fixed in [8d025b3](https://github.com/EnigmaCurry/d.rymcg.tech/pull/242/commits/8d025b3a3b5c5f48cc0d1507d689211485053c53)